src/material/schematics/ng-update/test-cases/v9/hammer-migration-v9.spec.ts

```diff
@@ -59,9 +59,12 @@ describe('v9 HammerJS removal', () => {
         import 'hammerjs';
       `);
 
-      await runMigration();
+      const {logOutput} = await runMigration();
 
       expect(tree.readContent('/projects/cdk-testing/src/main.ts')).not.toContain('hammerjs');
+      expect(logOutput).toContain(
+        'General notice: The HammerJS v9 migration for Angular components is not able to ' +
+        'migrate tests. Please manually clean up tests in your project if they rely on HammerJS.');
```

Comment:
"manually clean up tests in your project" -> "resolve any test issues in your project manually"

Reply:
Can we provide them any further information about how they are supposed to fix this tests? Is there at least a changelog w/ a BREAKING CHANGES entry that we can link them to?

Reply:
I intentionally used "clean up" here since it's not necessarily only test issues. e.g. one could import `hammer` manually and he should rework his test to not depend on that import.

I agree that it would be nice to link them to. We don't have it listed as a breaking change yet since it's just deprecated for v9. Maybe we should create an update guide though.

Reply:
re: "clean up", the need to "clean up" your project after the migration runs implies that the migration made a mess or left things in a dirty and undesirable state. That's why I was suggesting the "resolve any test issues" wording. It has a less negative implication.